files/css/element.css

```diff
@@ -13,8 +13,9 @@
     margin-top: -10px;
     margin-bottom: -10px;
     text-align: center;
-    padding: 0;
-    min-height: @user-number-size;
+    padding: 5px 0 0;
+    min-height: @user-number-size + 10px;
+    line-height: @user-number-size;
```

Comment:
if you do `line-height: @user-number-size + 10px` then you shouldn't even need `min-height`